modules/twinkleblock.js

```diff
@@ -465,6 +465,12 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 						})
 					};
 				}
+			},
+			templateSelection: function(choice) {
```

Comment:
This is just cool as fuck! 😎

Reply:
Looks like select2 has got an option for almost everything :)